bagelbids/tests/data/example_invalid.tsv

```diff
@@ -0,0 +1,5 @@
+participant_id	session_id	group
```

Comment:
ah yes. thanks for catching this. I agree, this isn't very clear right now. The main reason this file exists at all is because I need to pass "something" that is a .tsv and has the same name as the .json I really care about to the CLI. I'll add something to the README.